src/Symfony/Component/PropertyAccess/composer.json

```diff
@@ -20,7 +20,8 @@
         "symfony/property-info": "^6.4|^7.0|^8.0"
     },
     "require-dev": {
-        "symfony/cache": "^6.4|^7.0|^8.0"
+        "symfony/cache": "^6.4|^7.0|^8.0",
+        "symfony/var-exporter": "^6.4.1|^7.0.1|^8.0"
```

Comment:
to benefit from the fix in #52797

Reply:
PR not needed: these deprecations should be silenced on deps=low

Reply:
It's not a deprecation causing [the failure](https://github.com/symfony/symfony/actions/runs/15410625017/job/43361650031#step:9:2559):

```
There was 1 error:

1) Symfony\Component\PropertyAccess\Tests\PropertyAccessorTest::testGetValuePropertyThrowsExceptionIfUninitializedWithLazyGhost
Undefined array key 3

/home/runner/work/symfony/symfony/src/Symfony/Component/PropertyAccess/vendor/symfony/var-exporter/LazyGhostTrait.php:170
/home/runner/work/symfony/symfony/src/Symfony/Component/PropertyAccess/PropertyAccessor.php:436
/home/runner/work/symfony/symfony/src/Symfony/Component/PropertyAccess/PropertyAccessor.php:102
/home/runner/work/symfony/symfony/src/Symfony/Component/PropertyAccess/Tests/PropertyAccessorTest.php:1013
```

Reply:
Ah sorry, :+1: